### PR TITLE
fix:change mobile modal width

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

File: libs/web-components/src/components/modal/Modal.svelte [hunk 1]
```diff
@@ -126,7 +125,7 @@
     position: relative;
     background-color: #fff;
     z-index: 1001;
-    width: var(--width, 60ch);
+    width: 90%;
```
[the remark this answers is not in the document]
Comment (chrisolsen) [reply libs/web-components/src/components/modal/Modal.svelte:128]: @ravi-dio there is no issue it is just part of the requirements

[the remark this answers is not in the document]
Comment (chrisolsen) [reply libs/web-components/src/components/modal/Modal.svelte:128]: @ravi-dio this value is only for the modal on the mobile side so there is no variable defined at the current time.

**Review by bluekurinji — APPROVED** [review]

**Pull request merged**